### ISSUE TITLE
Increase the localization of the page

### ISSUE THREAD

**Issue by 3442853561** [servo/servo.org_2014-2020#36]
Increase the localization of the page

**Comment by 3442853561** [comment]
I have already submitted a pull request

**Comment by 3442853561** [comment]
#38

**Comment by KiChjang** [comment]
This is still not done yet for the Chinese localization page unfortunately, the CSS styling is completely off on the page.

**Comment by Coder206** [comment]
I also believe the localization menu styling should be improved.

Current:

![page-shot-2016-11-14 servo the parallel browser engine](https://cloud.githubusercontent.com/assets/7758155/20287975/f954e97e-aa9d-11e6-8fd2-e0b75a106eef.png)

**Comment by KiChjang** [comment]
I believe the style is following https://www.rust-lang.org. @Coder206 I'd like to hear your design proposals on how this should be improved.

**Comment by Coder206** [comment]
@KiChjang Correct. it just strikes me odd that everything has a uniform left and right padding but the localization is off to the side...

I made some renderings to demonstrate the current problem (in my opinion) and my proposal:

## Current:

![page-shot-2016-11-14 servo the parallel browser engine_current](https://cloud.githubusercontent.com/assets/7758155/20288514/f35d9b98-aaa1-11e6-8b58-1dd69b888bea.png)
The orange/yellow is just to outline the padding from the paragraph above, notice how the localization menu continues through it.

## Proposal:

![page-shot-2016-11-14 servo the parallel browser engine_proposal1](https://cloud.githubusercontent.com/assets/7758155/20288516/fb6c78b8-aaa1-11e6-89d3-ac66b8be31cb.png)
Notice that it matches with the menu below and it could potentially be combined in the footer with the other links.

**Comment by ys-chung** [comment]
The problem is that the version of Bootstrap used here is off-the-shelf, and the `.container` element of this version is much wider than what we needed, on the desktop. Therefore the "solution" that was used in servo.org is to only use 8/12 of the 12-grid system, and add margin the width of 2 grids to the left. 

The cleanest solution would be to [customize](https://getbootstrap.com/customize/) Bootstrap so that its `.container` maximum width is around where we need. And I'm working on that, will submit a pull request soon.

**Comment by tigercosmos** [comment]
I am going to translate to traditional Chinese

**Comment by Hikai** [comment]
I translated Korean and already submitted pull request. #56 
If that merge, I able to translation Japanese.
